ui/src/message_popup/author/edit_question_page.jsx

```diff
@@ -11,15 +11,67 @@ export default React.createClass({
   displayName: 'EditQuestionPage',
 
   propTypes: {
-    questionId: React.PropTypes.string.isRequired
+    questionId: React.PropTypes.string.isRequired,
+    loaded: React.PropTypes.bool,
+    allQuestions: React.PropTypes.object,
+    onReloadQuestions: React.PropTypes.func
+  },
+
+  componentDidMount(){
+    this.props.onReloadQuestions();
```

Comment:
Ah got it, great diagnosis and explanation.  Yeah I don't think they'd need to reload on that navigation if we update their values directly when changing the questions.  Let's talk it out in person.